tests/models/unit_cases_gas/source_sink/src/run_source_sink.py

```diff
@@ -52,12 +62,22 @@ class GasProblem(
     ESDLMixin,
     CollocatedIntegratedOptimizationProblem,
 ):
-    def path_goals(self):
-        goals = super().path_goals().copy()
-
-        return goals
+    """
+    Problem to check optimization behaviour for a producer, pipe, sink network.
+    """
 
     def times(self, variable=None) -> np.ndarray:
+        """
+        We shorten the timeseries to speed up the test
```

Comment:
suggest using 3rd person: "Shorten the time series ..."

Reply:
done